src/features/CreateEntities/ui/cinemas/CinemaCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC, useState } from 'react';
+
+import { Button } from '@mui/material';
+import { useQueryClient } from '@tanstack/react-query';
+
+import { useDeleteCinemaMutation } from 'entities/cinema/hooks/useDeleteCinema';
+import { useTranslation } from 'shared/hooks/i18nHook';
+import { Alert, Dialog } from 'shared/ui';
+
+interface CinemaCardProps {
+    id: string;
+    name: string;
+    city: string;
+    onEditClick: () => void;
+}
+
+export const CinemaCard: FC<CinemaCardProps> = ({ name, city, id, onEditClick }) => {
+    const { mutate: deleteCinema } = useDeleteCinemaMutation();
+    const [showDeleteDialog, setShowDeleteDialog] = useState(false);
+    const [mutationError, setMutationError] = useState('');
+
+    const queryClient = useQueryClient();
+    const { t } = useTranslation('common');
+
+    const onDelete = (cinemaId: string) => {
+        deleteCinema(cinemaId, {
+            onSuccess: () => {
+                queryClient.invalidateQueries({ queryKey: ['cinemas'] });
+                setShowDeleteDialog(false);
+            },
+            onError: () => {
+                setMutationError('This cinema is part of reservation an cannot be deleted');
```

Comment:
```suggestion
                setMutationError('This cinema is part of reservation and cannot be deleted');
```
Let's use translation